server/index.js

```diff
@@ -1,17 +1,24 @@
+const cors = require('cors');
 const express = require('express');
 const morgan = require('morgan');
 const port = 6336;
+const path = require('path');
 const app = express();
 
 app.use(morgan('dev'));
 app.use(express.urlencoded({
   extended: true,
 }));
+app.use(cors());
```

Comment:
At some point, maybe consider calling `cors()` as the second argument to the specific routes that you want to allow vs leaving your service/proxy open to all request made from anyone anywhere.